config/config.go

```diff
@@ -472,6 +472,7 @@ type Performance struct {
 	MaxMemory             uint64  `toml:"max-memory" json:"max-memory"`
 	ServerMemoryQuota     uint64  `toml:"server-memory-quota" json:"server-memory-quota"`
 	MemoryUsageAlarmRatio float64 `toml:"memory-usage-alarm-ratio" json:"memory-usage-alarm-ratio"`
+	AnalyzeMemoryQuota    uint64  `toml:"analyze-memory-quota" json:"analyze-memory-quota"`
```

Comment:
I suggested to remove `quota` config as we only tended to track memory consmuing for now.